Dokan/Dokan/Scenes/ProductDetails/Cell/ReviewerTableViewCell.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
Please merge develop and make sure the project is building. SwiftFormat should automatically fix the indentation here.